stream_alert/rule_processor/rules_engine.py

```diff
@@ -403,3 +418,36 @@ def process(cls, input_payload):
                     alerts.append(alert)
 
         return alerts
+
+    @classmethod
+    def get_intelligence(cls, intel_dir='threat_intel'):
+        """Load intelligence from csv.gz files into a dictionary
+
+        Args:
+            intel_dir (str): Location where stores compressed intelligence
+        """
+        threat_intel = ThreatIntel(intel_dir)
```

Comment:
Move the instantiation of `ThreatIntel` to after the `if ... return` check since the object doesn't even need created if the dict is already present